### PR TITLE
wcoss2 cycled ecflow archive and verification JJobs

### PR DESCRIPTION
**Pull request by lgannoaa**
**Description**

In this PR it will contain:
ecflow workflow verification and archive job scripts.
ecflow verification configuration file.
ecf/scripts/workflow_manager/cycled/enkfgdas/.gitignore - git ignore file for earc jobs
ecf/scripts/workflow_manager/cycled/enkfgdas/jenkfgdas_emc_earc_master.ecf - Master earc file
ecf/scripts/workflow_manager/cycled/gdas/atmos/jgdas_emc_vrfy.ecf - gdas verification ecflow job script
ecf/scripts/workflow_manager/cycled/gdas/jgdas_emc_arch.ecf - gdas archive ecflow job script
ecf/scripts/workflow_manager/cycled/gfs/atmos/jgfs_emc_vrfy.ecf - gfs verification ecflow job script
ecf/scripts/workflow_manager/cycled/gfs/jgfs_emc_arch.ecf - gfs archive ecflow job script
ecf/setup_ecf_links.sh - linking earc jobs
parm/config/config.ecfvrfy - verification configuration file for ecflow workflow GDAS/GFS vrfy jobs

closes: #720 

**Type of change**

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**

Tested with both rocoto workflow and ecflow workflow.

  
**Checklist**

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: ecf/setup_ecf_links.sh [hunk 1]
```diff
@@ -18,12 +18,31 @@ function link_master_to_fhr(){
   done
 }
 
+# Function that loops over group number and
+# creates link between the master and target
+function link_master_to_grp(){
+  tmpl=$1  # Name of the master template
+  grps=$2  # Array of forecast hours
+  for grp in ${grps[@]}; do
+    master=${tmpl}_master.ecf
```
Comment (WalterKolczynski-NOAA) [inline ecf/setup_ecf_links.sh:27]: This is the same for every loop and could be moved before the loop.

File: parm/config/config.ecfvrfy [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    export CUE2RUN=$QUEUE
+
+    export VBACKUP_FITS=24
+
+    export CONVNETC="NO"
+    if [ ${netcdf_diag:-".false."} = ".true." ]; then
+        export CONVNETC="YES"
+    fi
+
+    if [ $machine = "WCOSS_C" ]; then
```
Comment (WalterKolczynski-NOAA) [inline parm/config/config.ecfvrfy:50]: Can we use a case statement instead of all these else-ifs?

File: parm/config/config.ecfvrfy [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+fi
+
+#----------------------------------------------------------
+# VSDB STEP1, Verify Precipipation and Grid To Obs options
+#----------------------------------------------------------
+# All these call $VSDBSH
+
+if [ "$CDUMP" = "gfs" ] ; then
+    ddd=`echo $CDATE |cut -c 1-8`
+    #if [ $ddd -eq 5 -o $ddd -eq 10 ]; then export VSDB_STEP2 = "YES" ;fi
```
Comment (WalterKolczynski-NOAA) [inline parm/config/config.ecfvrfy]: Remove dead code

File: parm/config/config.ecfvrfy [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+    fi
+
+fi
+
+
+#-------------------------------------------------
+# Cyclone genesis and cyclone track verification
+#-------------------------------------------------
+
+export ens_tracker_ver=v1.1.15.5
```
Comment (WalterKolczynski-NOAA) [inline parm/config/config.ecfvrfy:144]: Can we standardize the installations for the TC tracker so we don't need this confusing definition @KateFriedman-NOAA ? Different versions, different paths; it's a mess.

File: parm/config/config.ecfvrfy [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+fi
+
+if [[ "$VRFYFSU" == "YES" && "$CDUMP" == "gfs" ]]; then
+
+    export GENESISFSU="$HOMEgfs/jobs/JGFS_ATMOS_FSU_GENESIS"
+fi
+
+if [[ "$RUNMOS" == "YES" && "$CDUMP" == "gfs" ]]; then
+
+    if [ $machine = "WCOSS_C" ] ; then
```
Comment (WalterKolczynski-NOAA) [inline parm/config/config.ecfvrfy]: Can we move these settings up to the other machine section instead? I also don't know that it matters if we define these only for gfs, if they aren't used anyway. In the above section, I would just either set RUNGFSMOSSH, or set RUNMOS to "NO", based on the machine.

Also question why there is a different script for each machine, but that's outside the scope of this PR.

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]

File: parm/config/config.ecfvrfy (hunk 4 above)
Comment (KateFriedman-NOAA) [reply parm/config/config.ecfvrfy:144]: My test version of config.vrfy for this branch on WCOSS2 consolidated the `HOMEens_tracker` (and `ens_tracker_ver`) settings to this:

```
export HOMEens_tracker=$BASE_GIT/TC_tracker/${tracker_ver}
```

I have been installing the newer copy of `TC_tracker` on each supported platform in a folder called `TC_tracker` with a subfolder (with the install) based on the version (`tracker_ver`). This matches how NCO installs packages on WCOSS2 (application/version). I also added `export tracker_ver=v1.1.15.5` to my `$target.ver` files.

See this on Cactus: `/lfs/h2/emc/global/save/emc.global/git/TC_tracker/v1.1.15.5`

My testing of this piece was put on hold to attend to WCOSS2 ops tasks so I haven't committed this back yet.

Comment (KateFriedman-NOAA) [reply parm/config/config.ecfvrfy:144]: On Cactus, see /lfs/h2/emc/global/noscrub/Kate.Friedman/expdir/devv16_192/config.vrfy (tracker section) for the consolidation I did.

Comment (lgannoaa) [reply parm/config/config.ecfvrfy:144]: At time this PR/issue created, the information in question was not available. We can modify it when now solution is tested.

**Comment by aerorahul** [comment]
Closing after discussions w/ @lgannoaa

**Pull request closed without merging**